profiles/CodingSpecies.json

```diff
@@ -1,5 +1,5 @@
 {
-  "image": "https://user-images.githubusercontent.com/70807500/155700104-38a53f35-2d41-4423-af94-53a3901d4f6d.png",
+  "image": "https://user-images.githubusercontent.com/70807500/174457276-0aadcf75-79a8-4217-843d-358ca0726d5d.png)![image](https://user-images.githubusercontent.com/70807500/174457276-0aadcf75-79a8-4217-843d-358ca0726d5d.png",
```

Comment:
It looks like there are two images here. Could you keep only one, please?


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient